tests/torch/nas/creators.py

```diff
@@ -133,9 +133,11 @@ def create_supernet(
     return ctrl.multi_elasticity_handler, model
 
 
-def create_single_conv_kernel_supernet(kernel_size=5, out_channels=1) -> Tuple[ElasticKernelHandler, NNCFNetwork]:
+def create_single_conv_kernel_supernet(
+    kernel_size=5, out_channels=1, padding=2
```

Comment:
Set `padding = 2` so that after enabling elastic kernel, this function can create the same supernet as the old one. Other modification of the padding value are for the same reason.